common/src/main/java/tachyon/LeaderInquireClient.java

```diff
@@ -65,6 +65,7 @@ private LeaderInquireClient(String zookeeperAddress, String leaderPath, TachyonC
     mZookeeperAddress = zookeeperAddress;
     mLeaderPath = leaderPath;
 
+    LOG.info("create new zookeeper client. address: {}", mZookeeperAddress);
```

Comment:
I think this is ok, because this should only print out a line per JVM, per unique zookeeper address. The constructor is only created once per unique zookeeper address, kept in a hashmap.